### PR TITLE
[#62] dicomweb path validation

### PR DESCRIPTION
**Pull request by red1408**

### CODE REVIEW

File: dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebValidation.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    private static String STUDIES = "studies";
+    private static String STUDIES_PATH = DICOMWEB_PATH + "\\/" + STUDIES;
+    private static String HEALTHCARE_API_ROOT = "https://healthcare.googleapis.com";
+
+    public static ValidationPattern DICOMWEB_ROOT_VALIDATION =
+            new ValidationPattern(Pattern.compile(DICOMWEB_PATH), "Google Healthcare Api dicomWeb root path");
+    public static ValidationPattern DICOMWEB_STUDIES_VALIDATION =
+            new ValidationPattern(Pattern.compile(STUDIES_PATH), "Google Healthcare Api dicomWeb studies path");
+
+    public static void validatePath(String path, ValidationPattern validation){
```
Comment (danielbeaudreau) [inline dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebValidation.java:21]: Can you please add a couple of unit tests for this class?

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java [hunk 2]
```diff
@@ -164,10 +164,11 @@ private static DicomRedactor configureRedactor(Flags flags) throws IOException{
     DestinationsConfig conf = new DestinationsConfig(destinationJsonInline, destinationsJsonPath);
     Map<DestinationFilter, IDicomWebClient> result = new LinkedHashMap<>();
     for (String filterString : conf.getMap().keySet()) {
+      String filterPath = StringUtil.trim(conf.getMap().get(filterString));
```
Comment (danielbeaudreau) [inline import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java:167]: Can we validate the filter paths at the start of execution as well?

Comment (red1408) [reply import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java:167]: Validation already happens in DicomWebClien/DicomWebClientJetty constructors, invoked as part of startup.

File: dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebValidation.java (hunk 1 above)
Comment (red1408) [reply dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebValidation.java:21]: Ok, added.

**Pull request merged**